src/listeners/onVoiceChannelJoin/jesus.ts

```diff
@@ -0,0 +1,25 @@
+import { Message, SnowflakeUtil, TextChannel, VoiceState } from "discord.js";
+import commands from "../../commands";
+
+const JESUS_MEMBER_ID = "191931417314328576";
+
+export default (oldState: VoiceState, newState: VoiceState) => {
+    if (newState.member?.id !== JESUS_MEMBER_ID) return;
```

Comment:
I'd suggest moving this to the `onVoiceChannelJoin` method so that you only call the `jesus()` method if jesus joined the channel